### PR TITLE
Use https for cldf.clld.org urls

### PR DESCRIPTION
**Pull request by ar-jan**
Fixes browser scrolling to url fragments.

### CODE REVIEW

**Comment by ar-jan** [comment]
Oh wait, I guess this may not be correct for schema URLs. But would be helpful for documentation links.

**Comment by xrotwang** [comment]
Yes, for better or worse, we are stuck with the term URIs using the `http` scheme.

**Comment by ar-jan** [comment]
I've now omitted the schema URLs from the replacement.

File: README.md [hunk 1]
```diff
@@ -138,10 +138,10 @@ to the CLDF Ontology. Each property also has a `rdf:label` which might also be d
 - In an ill-advised attempt to version the ontology, `v1.0` has been baked into the term
   URIs. While this may be a good idea in case of incompatible changes (e.g. if the semantics
   of a term changed), it presents an obstacle for initeroperability in case of backwards-compatible
-  changes. So starting with CLDF 1.1, we will keep `http://cldf.clld.org/v1.0/terms.rdf`
+  changes. So starting with CLDF 1.1, we will keep `https://cldf.clld.org/v1.0/terms.rdf`
```
Comment (xrotwang) [inline README.md:141]: This is the namespace URL and should stay with `http` as well.

File: README.md [hunk 2]
```diff
@@ -340,10 +340,10 @@ group *forms* into *cognate sets*, creating a [many-to-many relationship](https:
 between a `FormTable` and a `CognatesetTable`. 
 
 To make such relations explicit, the CLDF Ontology provides a set of
-[reference properties](http://cldf.clld.org/v1.0/terms.rdf#reference-properties).
+[reference properties](https://cldf.clld.org/v1.0/terms.rdf#reference-properties).
 
 Reference properties MUST be interpreted as foreign keys, e.g. a 
-`propertyUrl` `http://cldf.clld.org/v1.0/terms.rdf#languageReference`
+`propertyUrl` `https://cldf.clld.org/v1.0/terms.rdf#languageReference`
```
Comment (xrotwang) [inline README.md:346]: Ontology term. Keep `http`.

File: README.md [hunk 3]
```diff
@@ -357,11 +357,11 @@ specified for column `Col1` of a table with `url` `table1.csv` is equivalent to
        }
    ]
 ```
-assuming that the `LanguageTable` component has `url` `languages.csv` and a column `ID` with `propertyUrl` `http://cldf.clld.org/v1.0/terms.rdf#id`.
+assuming that the `LanguageTable` component has `url` `languages.csv` and a column `ID` with `propertyUrl` `https://cldf.clld.org/v1.0/terms.rdf#id`.
```
Comment (xrotwang) [inline README.md:360]: Ontology term. Keep http.

File: README.md [hunk 4]
```diff
@@ -357,11 +357,11 @@ specified for column `Col1` of a table with `url` `table1.csv` is equivalent to
...
 
 While spelling out foreign key constraints may feel cumbersome, it is still RECOMMENDED that metadata creators
 do so, to make the data compatible with CSVW tools. The foreign key constraints MUST be specified explicitly 
-if the referenced column does not have a `propertyUrl` `http://cldf.clld.org/v1.0/terms.rdf#id`.
+if the referenced column does not have a `propertyUrl` `https://cldf.clld.org/v1.0/terms.rdf#id`.
```
Comment (xrotwang) [inline README.md:364]: Ontology term. Keep http.

File: components/codes/README.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ This specification can be done by adding a codes table with the default descript
 [`CodeTable-metadata.json`](CodeTable-metadata.json).
 
 In a `StructureDataset`'s `ValueTable` the codes can be referenced using a foreign
-key with `propertyUrl` `http://cldf.clld.org/v1.0/terms.rdf#codeReference`.
+key with `propertyUrl` `https://cldf.clld.org/v1.0/terms.rdf#codeReference`.
```
Comment (xrotwang) [inline components/codes/README.md:19]: Ontology term. Keep http.

File: components/codes/description.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ This specification can be done by adding a codes table with the default descript
 [`CodeTable-metadata.json`](CodeTable-metadata.json).
 
 In a `StructureDataset`'s `ValueTable` the codes can be referenced using a foreign
-key with `propertyUrl` `http://cldf.clld.org/v1.0/terms.rdf#codeReference`.
+key with `propertyUrl` `https://cldf.clld.org/v1.0/terms.rdf#codeReference`.
```
Comment (xrotwang) [inline components/codes/description.md:19]: Ontology term. Keep http.

File: components/cognates/README.md [hunk 7]
```diff
@@ -17,7 +17,7 @@ Like cognates, partial cognates refer to a form in a wordlist. But to make it
 possible to annotate parts of a form, the segmentation of the form
 **must** contain morpheme boundaries,
 i.e. the `FormTable` must contain a column with `propertyUrl`
-`http://cldf.clld.org/v1.0/terms.rdf#segments` where the secondary separator
+`https://cldf.clld.org/v1.0/terms.rdf#segments` where the secondary separator
```
Comment (xrotwang) [inline components/cognates/README.md:20]: Ontology term. Keep http.

File: components/cognates/description.md [hunk 8]
```diff
@@ -17,7 +17,7 @@ Like cognates, partial cognates refer to a form in a wordlist. But to make it
 possible to annotate parts of a form, the segmentation of the form
 **must** contain morpheme boundaries,
 i.e. the `FormTable` must contain a column with `propertyUrl`
-`http://cldf.clld.org/v1.0/terms.rdf#segments` where the secondary separator
+`https://cldf.clld.org/v1.0/terms.rdf#segments` where the secondary separator
```
Comment (xrotwang) [inline components/cognates/description.md:20]: Ontology term. Keep http.

File: components/contributions/description.md [hunk 9]
```diff
@@ -29,5 +29,5 @@ In contrast, the [World Loanword Database](https://wold.clld.org/) contains voca
 contributed by an expert on each language and its history. In this case, therefore,
 the contributions are linked to each language.
 
-Thus, for WALS Online one would add a column with `propertyURL` `http://cldf.clld.org/v1.0/terms.rdf#contributionReference`
+Thus, for WALS Online one would add a column with `propertyURL` `https://cldf.clld.org/v1.0/terms.rdf#contributionReference`
```
Comment (xrotwang) [inline components/contributions/description.md:32]: Ontology term. Keep http.

**Review by xrotwang — CHANGES_REQUESTED** [review]
Basically, only the property URLs which are used as link targets for a **labeled** link might be changed to `https`. And even then, I could imagine people using "copy link" functionality of their browsers intending to get proper property URLs. So overall, I fear this change might introduce more confusion than convenience.

**Comment by ar-jan** [comment]
Oh, I missed the backtick variants. But given this,

> And even then, I could imagine people using "copy link" functionality of their browsers intending to get proper property URLs.

I guess I'd better just leave this alone and close, sorry for the noise.

**Comment by xrotwang** [comment]
Some noise is appreciated. Now we have something to refer to whenever this discussion comes up again :)

**Pull request closed without merging**